### DISCUSSION TITLE
Send message to Telegram Group's Topic

### DISCUSSION THREAD

**Discussion by stecya** [containrrr/shoutrrr#316] [Q&A] [answered]
Hi,

I recently converted my group chat to support topics. By default shoutrrr will send the general one, however, I would like to send to the particular one. Is it possible to do that?

Based on Telegram API one needs to provide "message_thread_id" in order to send to a particular topic

**Comment by piksel** [comment, marked as answer]
Hm, no I don't think so right now, but that should be pretty easy to add!

**Reply by stecya** [reply]
Would really appreciate if you can add this feature. Many thanks.

**Reply by oliveratgithub** [reply]
Has this been added in the meantime? The Docs don't have any example for a Telegram-Chat / Channel ID PLUS Topic ID...

Edit: looks like a relevant PR is ready at #521
> preserve the existing per-chat chat_id:thread_id syntax and give it precedence

**Comment by Austinsuyoyo** [comment]
It's can use generic like below in docker-compose.yaml

use `%3A` replace `:`
```
- WATCHTOWER_NOTIFICATION_URL=generic://api.telegram.org/bot{xxx}%3A{token}/sendMessage
- 'WATCHTOWER_NOTIFICATION_TEMPLATE={{if .}}{"text":"{{range .}}{{.Message}}\n{{end}}","chat_id":"{id}","message_thread_id":"5"}{{end}}'
```
